spec/support/edit_conflicts_shared_examples.rb

```diff
@@ -29,8 +29,16 @@ def conflict_warning
     end
   end
 
-  context "when another user has updated the record in the meantime" do
-    let(:email_1) { "someone@example.com" }
+  specify "an update wo/ original_updated_at doesn't say anything about edit conflicts", js: true do
```

Comment:
```suggestion
  specify 'an update wo/ original_updated_at doesn't say anything about edit conflicts', js: true do
```

Reply:
I am not sure about committing this suggestion, the string already contains a `'` in `doesn't`

Reply:
oh you're right. use `doesn\'t` instead?